### PR TITLE
Make Tester::new async

### PR DESCRIPTION
**Pull request by ShaddyDC**

### CODE REVIEW

File: storages/memory-storage/tests/memory_storage.rs [hunk 1]
```diff
@@ -1,11 +1,13 @@
+use async_trait::async_trait;
 use {gluesql_core::prelude::Glue, gluesql_memory_storage::MemoryStorage, test_suite::*};
```
Comment (ever0de) [inline storages/memory-storage/tests/memory_storage.rs:2]: Please group `use` statements

```suggestion
use {gluesql_core::prelude::Glue, gluesql_memory_storage::MemoryStorage, test_suite::*}, async_trait::async_trait;
```

File: storages/shared-memory-storage/tests/shared_memory_storage.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+use async_trait::async_trait;
```
Comment (ever0de) [inline storages/shared-memory-storage/tests/shared_memory_storage.rs:1]: ditto

File: storages/sled-storage/tests/sled_storage.rs [hunk 3]
```diff
@@ -1,11 +1,13 @@
+use async_trait::async_trait;
```
Comment (ever0de) [inline storages/sled-storage/tests/sled_storage.rs:1]: ditto

**Review by ever0de — COMMENTED** [review]
Also change the storage in `pkg/javascript/*` (see CI)

**Comment by ShaddyDC** [comment]
Oops, I didn't mean to remove the other review requests by requesting a rereview. Anyway, those should be addressed @ever0de

**Review by panarch — APPROVED** [review]
Looks all nice! Thanks a lot for the contribution 👍 👍 👍

**Pull request merged**